sotodlib/site_pipeline/update_det_cal.py

```diff
@@ -720,14 +785,23 @@ def handle_result(result: CalRessetResult, cfg: DetCalCfg) -> None:
         msg = result.fail_msg
         if msg is None:
             msg = "unknown error"
+        if 'sotodlib.core.metadata.loader.LoaderError' in msg:
+            logger.error(f"obs_id {obs_id} failed due to medatada loader "
+                         "error, try again later")
+            return
         add_to_failed_cache(cfg, obs_id, msg)
         return
 
     logger.info(f"Adding obs_id {obs_id} to dataset")
     rset = ResultSet.from_friend(result.result_set)
-    write_dataset(rset, cfg.h5_path, obs_id, overwrite=True)
+    h5_path = cfg.h5_path
+    if cfg.h5_unix_digits:
+        name, ext = os.path.splitext(cfg.h5_path)
+        unixtime = obs_id.split('_')[1][:cfg.h5_unix_digits]
+        h5_path = f"{name}_{unixtime}{ext}"
```

Comment:
Question, no edits required. This is unrelated to hwpss subtraction but you're adding it here to minimize the size of metadata archive files and to make it more in alignment with some of the other metadata formatting?